modules/ROOT/pages/notifications/all-notifications.adoc

```diff
@@ -2183,9 +2320,39 @@ END AS standardsName
 ======
 =====
 
-.Using an unescaped variable named `contains` in addition or subtraction operations within a `WHEN` operand in a simple `CASE` expression
+.Using an unescaped variable named `contains` for relationships in addition or subtraction operations within a `WHEN` operand in a simple `CASE` expression
```

Comment:
The variable `contains` in the example bind to an INTEGER (the number of relationships in the path) and that is primarily a characteristics of the specific examples. 

As more abstract example would be:
```
WITH 2 AS x, 1 AS contains
RETURN CASE x
  WHEN contains + 1 THEN "foo"
  ELSE "bar"
END AS check
```
This does not involve any nodes and relationships — to make the point that nodes and relationships are not integral to this deprecations.